service_notification/src/main/kotlin/org/thechance/service_notification/plugins/StatusPage.kt

```diff
@@ -23,4 +24,8 @@ fun StatusPagesConfig.handleExceptions() {
         call.respond(HttpStatusCode.NotFound, listOf(cause.message?.toInt()))
     }
 
+    exception<InternalServerErrorException> { call, cause ->
+        call.respond(HttpStatusCode.InternalServerError, listOf(cause.message?.toInt()))
+    }
+
```

Comment:
To cut the duplication for exception handlers, you can check the identity-service